pkgs/development/libraries/atk/default.nix

```diff
@@ -18,7 +19,8 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev" ];
 
   nativeBuildInputs = [ meson ninja pkg-config gettext gobject-introspection glib ]
-    ++ lib.optional stdenv.hostPlatform.isDarwin fixDarwinDylibNames;
+    ++ lib.optional stdenv.hostPlatform.isDarwin fixDarwinDylibNames
+    ++ lib.optional withIntrospection gobject-introspection;
```

Comment:
Isn't it included unconditionally in the default list above??

Reply:
Good catch. Due to the build flag it's unused, but this doesn't look correct.

Reply:
Fixed now and it still builds :).

Reply:
Well, it's more interesting if rev-deps still build, I guess!